DuckDuckGo/AppDelegate.swift

```diff
@@ -33,13 +33,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     private struct ShortcutKey {
         static let clipboard = "com.duckduckgo.mobile.ios.clipboard"
     }
-    
-    static var shared: AppDelegate {
-        // swiftlint:disable force_cast
-        return UIApplication.shared.delegate as! AppDelegate
-        // swiftlint:enable force_cast
-    }
```

Comment:
As a part of exploring different options for keeping the tap link destination property in one place, I noticed that this `shared` property of `AppDelegate` isn't used any more. I've removed it, but let me know if this should stay.